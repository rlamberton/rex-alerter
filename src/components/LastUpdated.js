function LastUpdated(props) {
    return <div id='lastUpdated'>Last updated at {props.time}</div>
}

export default LastUpdated;
