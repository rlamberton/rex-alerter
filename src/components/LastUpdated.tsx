import React from 'react';
import LastUpdatedPropsType from '../types/LastUpdatedPropsType';

const lastUpdated : React.FC<LastUpdatedPropsType> = function LastUpdated(props) {
    return <div id='lastUpdated' data-testid='alert-list-last-updated'>Last updated at {props.time}</div>
}

export default lastUpdated;
