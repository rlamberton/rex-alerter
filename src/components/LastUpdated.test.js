import { render } from '@testing-library/react';
import LastUpdated from './LastUpdated';

test('check snapshot', () => {
    const { asFragment } = render(<LastUpdated time="10:30" />);
    expect( asFragment() ).toMatchSnapshot();
});

