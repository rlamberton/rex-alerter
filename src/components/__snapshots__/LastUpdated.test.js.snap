// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="alert-list-last-updated"
    id="lastUpdated"
  >
    Last updated at 10:30
  </div>
</DocumentFragment>
`;
